pytrustnfe/nfse/paulistana/templates/EnvioLoteRPS.xml

```diff
@@ -34,12 +34,12 @@
     <ISSRetido>{{ rps.iss_retido | default('false') }}</ISSRetido>
     {% if rps.tomador.tipo_cpfcnpj != 3 %}
     <CPFCNPJTomador>
-      {% if rps.tomador.tipo_cpfcnpj == 1 -%}
-      <CPF>{{ rps.tomador.cpf_cnpj }}</CPF>
-      {% endif %}
-      {% if rps.tomador.tipo_cpfcnpj == 2 -%}
-      <CNPJ>{{ rps.tomador.cpf_cnpj }}</CNPJ>
-      {% endif %}
+        {% if rps.tomador.tipo_cpfcnpj == 1 -%}
+           <CPF>{{ rps.tomador.cpf_cnpj }}</CPF>
+        {% endif %}
+        {% if rps.tomador.tipo_cpfcnpj == 2 -%}
+           <CNPJ>{{ rps.tomador.cpf_cnpj }}</CNPJ>
+        {% endif %}

```

Comment:
:nail_care: I'm afraid that all changes in this file would be unnecessary. :grimacing:

Reply:
I had to do these changes so, when we open a PR for the `master` branch on the original project, these files won't enter it :smile:  When I used the `git diff upstream/master resolve-fork2` command, I found out that (for some reason) this file had a different indentation than the upstream one :sweat_smile: